csharp/src/Drivers/Apache/Spark/README.md

```diff
@@ -41,6 +41,11 @@ but can also be passed in the call to `AdbcDatabase.Connect`.
 | `adbc.apache.statement.batch_size` | Sets the maximum number of rows to retrieve in a single batch request. | `50000` |
 | `adbc.apache.statement.polltime_ms` | If polling is necessary to get a result, this option sets the length of time (in milliseconds) to wait between polls. | `500` |
 | `adbc.apache.statement.query_timeout_s` | Sets the maximum time (in seconds) for a query to complete. Values can be 0 (infinite) or greater than zero. | `60` |
+| `ssl` | If ssl needs to enabled or not | `false` |
+| `allow_self_signed` | If self signed ssl certificate needs to be allowed or not  | `false` |
+| `allow_hostname_mismatch` | If hostname mismatch is allowed for ssl | `false` |
```

Comment:
`allow_self_signed` and `allow_hostname_mismatch` are already defined in the `adbc.spark.tls_options`. No need to redefine them.

You could add the `tls`, or `enable` to `adbc.spark.tls_options` to explicitly enable tls/ssl?